index.html

```diff
@@ -9,33 +9,32 @@
   <section class="banner" data-adaptive-background='1' data-ab-css-background='1'>
     <div class="wrapper">
       <h1>{{ site.title }}</h1>
-      <p>{{ site.description }}</p>
+        <p>{{ site.description }}</p>
```

Comment:
Why have you changed indentation here?

Also this is a good example of a change that should *definitely* have been a separate commit and quite possibly a separate Pull Request. Right now as reviewer i have only an all or nothing choice to merge or not merge this PR ;-)